### DISCUSSION TITLE
Move email confirmation from Supabase to backend

### DISCUSSION THREAD

**Discussion by c-obsesie** [supabase/supabase#22212] [Questions] [answered]
Is there a way to shift email confirmation from Supabase to my own backend?

Mind that i use Authentication option of supabase

**Comment by GaryAustin1** [comment, marked as answer]
Yes.  You can use https://supabase.com/docs/reference/javascript/auth-admin-generatelink to get tokens and url links to use in your own emails.

**Reply by c-obsesie** [reply]
thank you
